scripts/perftune.py

```diff
@@ -1258,10 +1313,12 @@ def names():
  --nic NIC       - default: eth0
  --cpu-mask MASK - default: all available cores mask
  --tune-clock    - default: false
+ --tune-rxqueues    - default: false
```

Comment:
We should also add the ability to override the amount of Rx queues: the default should be 1 - per IRQ CPU as in this patch and a user should be able to set the total number of RX_QUEUEs he/she wants that then will be configured and caped by the max HW supported value as per `ethtool -l`.